extensions/wikibase/src/org/openrefine/wikibase/updates/MediaInfoEdit.java

```diff
@@ -247,7 +255,18 @@ public MediaInfoIdValue uploadNewFile(WikibaseDataEditor editor, MediaFileUtils
 
         response.checkForErrors();
 
+        List<String> filenames = new ArrayList<String>();
+        filenames.add(fileName);
```

Comment:
You could use `Collections.singletonList(fileName)` instead, as you're building a list that isn't intended to be modified and only ever contain a single element. Not a big deal, but makes the code a bit simpler, clearer and marginally more efficient.

Reply:
Done.